logrotate.spec.in

```diff
@@ -1,7 +1,7 @@
 Summary: Rotates, compresses, removes and mails system log files
 Name: logrotate
 Version: @VERSION@
-Release: 1%{?dist}
+Release: 2%{?dist}
```

Comment:
Addressed comments and re-pushed.  Thanks for those.

Why not convert tabooext then to a list of globs?  We're already using `fnmatch()` so it would be relatively trivial... Of course, the `tabooext` keyword would have to be renamed `tabooglob` or something similar... And that would break existing configs.

Or `tabooext <str>` could be handled as `tabooglob *\.<str>` when it was seen to provide backwards compatibility.